bindings/python/tests/test_triplestore.py

```diff
@@ -0,0 +1,98 @@
+from dlite.triplestore import en, Literal, Triplestore, RDF, RDFS, XSD, OWL
+
+
+# Test namespaces
+assert str(RDF) == "http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+assert RDF.type == "http://www.w3.org/1999/02/22-rdf-syntax-ns#type"
+
+# Test RDF literal
```

Comment:
Many of these variables are not very descriptive, consider renaming. E.g. l1 = literal_helloworld

Reply:
The comment string is not very helpful 'testing literals' can be guessed by context. Consider adding more description. There are many different cases: l1,l2,l3,... consider adding comments as to why we are testing all these cases and what they are

Reply:
Thanks for the review. 

This is just a test script, where the comments are more ment for grouping the different types of tests rather than describing them. I underlined them to make that more clear. Do you feel more documentation is needed after reading the documentation of the triplestore module?

Regarding variable names there exists many opinions. My preferences are based on the following considerations:
- variable names should be concise and easy understandable from their context
- the name should describes what the variable is, rather than how it is used
- short code lines are easier to read than long code lines (especially if you have a tendency to dyslexia) - hense avoid unnecessary long variable names
- don't reuse the same variable name for different things

Hence, I don't think that `l1` is necessary a bad variable name, since its meaning is rather clear from the context. It is only used in a short code section, clearly separated by blank lines, where the first line
```python
l1 = Literal("Hello world!")
```
makes it easy to see that it is a literal. An alternative name could be `literal1`, but I think that `l` is a recognisable abbreviation of "literal" given the context. The number `1` is added because we are instantiating several literals and don't want to use the same name for them. It also emphasises that `l1` is just an instance.

Personally I think a name like `literal_helloworld` is to long and just make the code harder to read. What is interesting is that we are testing a literal, not that its value is "Hello world!". 

I think this is a very good issue to discuss and comment on, since people have different preferences and there are no absolute true or false, except for that consistency counts! Hence, we should try to follow the same practice within the team...